IFC 4.3/UT_Alignment_12d_1/UT_Alignment_12d_1.ifc

```diff
@@ -0,0 +1,42 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition[]'), '2;1');
+FILE_NAME ('UT_Alignment_12d_1.ifc', '2020-12-18T23:28:33', ('ljg'), ('12d Solutions - Ranch'), 'IFC Engine rev 1257', '12d Model 15.0 Alpha 408 [DEVELOP] INT Build 18-12-2020', '');
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+DATA;
+#1 = IFCPROJECT('2LJesmETv6W9jWXnI$q1Aj', #2, 'UT Alignment 12d 1', '', $, $, $, (#12), #7);
+#2 = IFCOWNERHISTORY(#3, #6, $, .ADDED., $, $, $, 1608294513);
+#3 = IFCPERSONANDORGANIZATION(#4, #5, $);
+#4 = IFCPERSON($, $, 'ljg', $, $, $, $, $);
+#5 = IFCORGANIZATION($, '12d Solutions - Ranch', '12d Solutions - Ranch', $, $);
+#6 = IFCAPPLICATION(#5, '15.0 Alpha 408 [DEVELOP] INT Build 18-12-2020', '12d Model', '12d Model');
+#7 = IFCUNITASSIGNMENT((#8, #9, #10, #11));
+#8 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#9 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#10 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#11 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#12 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #14, #18);
+#13 = IFCLOCALPLACEMENT($, #14);
+#14 = IFCAXIS2PLACEMENT3D(#15, #16, #17);
+#15 = IFCCARTESIANPOINT((0., 0., 0.));
+#16 = IFCDIRECTION((0., 0., 1.));
+#17 = IFCDIRECTION((1., 0., 0.));
+#18 = IFCDIRECTION((0., 1., 0.));
+#19 = IFCSITE('2iTRVf0N96fejtvVjbiFZR', #2, 'Default Site', 'Description of Default Site', $, #13, $, $, .ELEMENT., $, $, $, $, $);
+#20 = IFCRELAGGREGATES('1YL2QP0e55ivtEE6H_3Mm4', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#19));
+#21 = IFCFACILITY('09_nT1I4r0nup4KFCG4Dt7', #2, 'SA  no VG', 'Description of SA  no VG', $, #13, $, $, .ELEMENT.);
+#22 = IFCRELAGGREGATES('3Edrkwm61CrO1NEpyuWmSU', #2, 'SiteContainer', 'SiteContainer For All Models', #19, (#21));
+#23 = IFCRELCONTAINEDINSPATIALSTRUCTURE('2VLpKdG951IPpTkWgVrw_A', #2, 'Building Container', 'BuildingContainer for Elements', (#24), #21);
+#24 = IFCALIGNMENT('3TsZtwmXbER8Q1PJcHGYg2', $, 'RS01 no VG', '12d Alignment String', 'clothoid', $, $, $, $);
+#25 = IFCRELAGGREGATES('17txXWW0189Rb48wJLlvZl', #2, '', '', #24, (#26));
+#26 = IFCALIGNMENTHORIZONTAL('1aYHKEAr9ANuVigr7mUQoM', #2, 'Horizontal Alignment', '12d Horizontal Alignment', $, $, $, 0., (#28, #30, #32));
+#27 = IFCRELNESTS('0S6tA9lzrFWwczpMhigqbD', #2, '', '', #26, (#28, #30, #32));
+#28 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #29, IFCPLANEANGLEMEASURE(0.), 0., 0., IFCPOSITIVELENGTHMEASURE(48.5550856436632), $, .LINE.);
+#29 = IFCCARTESIANPOINT((0., 0.));
+#30 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #31, IFCPLANEANGLEMEASURE(0.), 150., 150., IFCPOSITIVELENGTHMEASURE(99.118453223473), $, .CIRCULARARC.);
```

Comment:
more a side note to improve documentation @SergejMuhic 
in RC2 documentation for [IFCALIGNMENTHORIZONTALSEGMENT ](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/link/ifcalignmenthorizontalsegment.htm) the definition of start and end curvature does not have an instruction on how to interpret positive and negative values in terms of CCW or CW. In this example it asumes that CCW is positve (also my assumption) but we should state it explicitly.

Reply:
Agree, if we document this than at least follow the idea behind IfcCircle, i.e. CCW if positive and CW if negative. I expect all agree on this but it is good to make it explicit.